frontend/public/components/edit-yaml.jsx

```diff
@@ -329,7 +329,14 @@ const EditYAML_ = connect(stateToProps)(
     }
 
     save() {
+      const { create, onSave } = this.props;
       let obj;
+
+      if (!create && onSave) {
```

Comment:
Why do we need to check `!create`?

Reply:
I was specifically passing in a save function, not intending to use for `create`, but I suppose I could not do the create check.

Reply:
For your use, but there's nothing about the prop that is specific to edit vs save. It'd be surprising to have `onSave` and not have it called because another prop is set to a certain value.

Reply:
ok, removed `!create` check